media-gfx/blender/blender-9999.ebuild

```diff
@@ -283,39 +310,49 @@ src_configure() {
 		-DWITH_BULLET=$(usex bullet)
 		-DWITH_CODEC_FFMPEG=$(usex ffmpeg)
 		-DWITH_CODEC_SNDFILE=$(usex sndfile)
-		-DWITH_CPU_CHECK=no
```

Comment:
Why did you remove this option?
This makes it hardcoded so that only cpu with sse4_2 can launch Blender even if you we don't compile with any sse4_2 instructions. So far this is only used for the official releases as they bumped the x86_64 `march` version.

If Blender starts to hard code in sse4_2 instructions in the future, it is better to specify these hard CPU requirements in the ebuild.